src/azure-cli/azure/cli/command_modules/storage/__init__.py

```diff
@@ -241,6 +246,19 @@ def handler(ex):
 
         return handler
 
+    @classmethod
+    def account_key_exception_handler(cls):
+        def handler(ex):
+            from azure.common import AzureException
+            from azure.core.exceptions import ClientAuthenticationError
+
+            if isinstance(ex, AzureException) and 'incorrect padding' in ex.args[0].lower():
+                raise CLIError('incorrect usage: the given account key may be not valid.')
```

Comment:
Is it possible that connection-string or sas-token throws an 'incorrect padding' exception, not limited to account key?